ui/app/pages/swaps/view-quote/view-quote-price-difference.js

```diff
@@ -45,9 +51,9 @@ export default function ViewQuotePriceDifference(props) {
     )
   }
 
-  const shouldShowPriceDifferenceWarning =
+  const shouldShowPriceDifferenceWarning = /*
     ['high', 'medium'].includes(priceSlippage.bucket) ||
```

Comment:
@darkwing this was added for your local development purposes, and should be removed